target/linux/kirkwood/image/Makefile

```diff
@@ -42,6 +42,13 @@ define Device/checkpoint_l-50
 endef
 TARGET_DEVICES += checkpoint_l-50
 
+define Device/globalscale_sheevaplug
+  DEVICE_TITLE := Globalscale Sheevaplug
```

Comment:
As you see in the file, DEVICE_TITLE has been replaced by DEVICE_VENDOR and DEVICE_MODEL ...

Reply:
Fixed.